torchgeo/datasets/utils.py

```diff
@@ -737,3 +737,23 @@ def percentile_normalization(
         (img - lower_percentile) / (upper_percentile - lower_percentile + 1e-5), 0, 1
     )
     return img_normalized
+
+
+def path_is_vsi(path: str) -> bool:
```

Comment:
Any idea what VSI stands for? Wondering if we should call this vfs instead.

Reply:
"Virtual Files System Interface" is the closest I can find to an explanation of the abbreviation. Could also be a play on URI? 😅

What I do like about keeping VSI, is communicating that the intention is to support _GDAL_ virtual file systems, not _any_ virtual filesystem.